src/Main.java

```diff
@@ -1,8 +1,73 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.printf("Hello and welcome!");
-        for (int i = 1; i <= 5; i++) {
+        System.out.println("Задание 1:");
+        int i = 1;
+        for (; i <= 10; i++) {
```

Comment:
Переменную лучше, конечно, объявить в самом цикле:
```suggestion
        for (int i = 1; i <= 10; i++) {
```